lib/workos/client.rb

```diff
@@ -86,7 +86,7 @@ def user_agent
       ].join('; ')
     end
 
-    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/CyclomaticComplexity:
```

Comment:
can we get rid of these cops? they are so annoying.

Reply:
Yes, but I don't think this is the PR for it.

Reply:
oh yeah sorry, that was mostly me griping (should have specified that i didn't intend for you to do that in this PR).